agenta-cli/agenta/cli/variant_commands.py

```diff
@@ -149,7 +151,22 @@ def add_variant(
         else:
             click.echo(click.style(f"Error while adding variant: {ex}", fg="red"))
         return None
+
+    # Get user profile
+    user_id = client.retrieve_user_id(host, api_key)
```

Comment:
we should run this line inside the conditional tracing_enabled

Reply:
Thank you!